Week2/enes-javascript2-homework.js

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Hi Enes! You should have changed the files in the `Week2/homework` folder, that way running the test will also work.  

The first should be written as 

```
function doubleOddNumbers(numbers) {
  const doubleNumbers = numbers.filter(number => number % 2 === 1).map(number => number * 2);
  return doubleNumbers;
}
```

If you run `npm run test-week2` you will see 1 test passed.